### PR TITLE
New documentation with SNMP configuration examples

### PR DESCRIPTION
**Pull request by FTBZ**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

- New documentation page with some SNMP configuration examples

### CODE REVIEW

File: doc/Support/SNMP-Configuration-Examples.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+source: Support/SNMP-Configuration-Examples.md
+
+# SNMP configuration examples
+
+Table of Content:
+- [Devices](#devices)
+    - [Cisco](#cisco)
+        - [Adaptive Security Appliance (ASA)](#adaptive-security-appliance-asa)
+        - [IOS](#ios)
+        - [IOS](#ios)
```
Comment (f0o) [inline doc/Support/SNMP-Configuration-Examples.md]: This seems doubled

File: doc/Support/SNMP-Configuration-Examples.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+## Operating systems
+### Linux (snmpd)
+
+Replace your snmpd.conf file by the example below and edit it with appropriate community in "RANDOMSTRINGGOESHERE".
+
+```
+vi /etc/snmp/snmpd.conf
+```
+
+```
```
Comment (murrant) [inline doc/Support/SNMP-Configuration-Examples.md]: Possibly point to the file in git so we don't miss this if the file needs to be updated.  Anyone agree/disagree?

File: doc/Support/SNMP-Configuration-Examples.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+### Cisco
+#### Adaptive Security Appliance (ASA)
+1. Launch ASDM and connect to your device
+2. Go to Configuration > Management Access > SNMP
+3. Add your community string
+4. Add in the "SNMP Host Access List" section your LibreNMS server IP address
+5. Click Apply and Save
+
+#### IOS
```
Comment (murrant) [inline doc/Support/SNMP-Configuration-Examples.md]: Might as well combine all the IOS ones, I don't see any reason they would need to be separate.

**Comment by FTBZ** [comment]
Requested changes have been made, only the link from or to the other page needs to be discussed more.

**Comment by f0o** [comment]
!trigger ci

**Comment by laf** [comment]
Was just missing adding this new link to the TOC as per the docs: http://docs.librenms.org/Developing/Creating-Documentation/

I've pushed a commit to add this so will merge once tests complete.

**Pull request merged**